### ISSUE TITLE
Is there a way to disable unchecking of application cookies?

### ISSUE THREAD

**Issue by Karlinator** [XcooBee/xcoobee-cookie-kit#93]
This is a pretty typical behaviour (and in line with GDPR) that strictly necessary cookies can't be disabled. For my project, for instance, very basic parts of the site does not work without certain cookies, and so I don't intend to provide the option of turning them off. That would, imo, be pretty pointless. If you can't tolerate those cookies then I don't have a site you can use anyway.

What is the intended way to communicate this to the user? I can't seem to find an option to disable the 'Required' checkbox apart from removing it from the dialog entirely but that seems not ideal. Is this even supported? If not, I'd like to request that feature.

Basically what I want is the "Required" checkbox (that is, application cookies) to be greyed out and permanently enabled.

I'm using the ReactJS component if that's relevant.

**Comment by Bilal-S** [comment]
Karl:
It should be possible to include this as option.
Currently you will have to show in your cookie declaration that application cookies will always be set and make your cookie handler behave accordingly.

**Comment by Karlinator** [comment]
Yeah, currently this is what I do. Tell the user that application cookies will be enabled regardless and then don't deactivate them. That is kind of confusing, however, as the user is able to uncheck them without that actually doing anything.

I'll be quite happy if this is added!

**Comment by Bilal-S** [comment]
@Karlinator 
bundle 1.0.14 has react-cookie-kit version 1.0.21 which changes behavior and always loads application/required cookies.
User cannot unchek.
